net/miniupnpd/Makefile

```diff
@@ -25,7 +25,7 @@ PLIST_FILES=	etc/miniupnpd.conf.sample \
 		man/man8/miniupnpd.8.gz \
 		sbin/miniupnpd
 
-OPTIONS_DEFINE=			CHECK_PORTINUSE IPV6 LEASEFILE UPNP_IGDV2 \
+OPTIONS_DEFINE=			PF_FILTER_RULES CHECK_PORTINUSE IPV6 LEASEFILE UPNP_IGDV2 \
```

Comment:
doesn't look like this serves a functional change and we set "PF" through tools.git:

config/22.7/make.conf:net_miniupnpd_SET=                CHECK_PORTINUSE PF